packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts

```diff
@@ -26,17 +26,9 @@ export const AlertConsumers = {
 export type AlertConsumers = typeof AlertConsumers[keyof typeof AlertConsumers];
 export type STATUS_VALUES = 'open' | 'acknowledged' | 'closed' | 'in-progress'; // TODO: remove 'in-progress' after migration to 'acknowledged'
 
-export const mapConsumerToIndexName: Record<AlertConsumers, string | string[]> = {
-  apm: '.alerts-observability-apm',
-  logs: '.alerts-observability.logs',
-  infrastructure: '.alerts-observability.metrics',
-  observability: '.alerts-observability',
-  siem: '.alerts-security.alerts',
-  uptime: '.alerts-observability.uptime',
-};
-export type ValidFeatureId = keyof typeof mapConsumerToIndexName;
+export type ValidFeatureId = AlertConsumers;
 
-export const validFeatureIds = Object.keys(mapConsumerToIndexName);
```

Comment:
Besides deleting the map itself, this PR actually fixes the names of the indices, particularly `.alerts-observability-apm` -> `.alerts-observability.apm.alerts`.
The current rule data client in the master branch writes to `.alerts-observability.apm.alerts`.

Reply:
actually... :nerd_face: `.alerts-observability.apm.alerts-default` which is backed by `.alerts-observability.apm.alerts-default-000001`